app/models/user.rb

```diff
@@ -3,7 +3,6 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
why did you comment out 'omniauthable'? that's what used for google authentication, so we'll need it

Reply:
If you see below(line 31) there is :omniauthable being called. Line 2 and 3 are auto generated by default and we can remove it.